pack.ps1

```diff
@@ -0,0 +1,165 @@
+param([string] $configuration = "Release")
+
+$ErrorActionPreference = 'Stop'
+$framework = "net472"
+$packageVersionFilePath = ".\package_version_temp.txt" # build.ps1 saves the package version to this temp file
+$packageVersion = Get-Content -Path $packageVersionFilePath
```

Comment:
Check existence before `Get-Content`. Hint user to run `build.ps` if it does not exists.